### PR TITLE
Feature/login functionality

### PR DESCRIPTION
**Pull request by paulinakujawa91**

### CODE REVIEW

File: locators/home_page_locators.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from selenium.webdriver.common.by import By
+
```
Comment (tomaszsobczak92) [inline locators/home_page_locators.py:2]: Can we be consistent throughout the solution and leave two blank lines after imports and before the class definition?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+def print_hi(name):
+    # Use a breakpoint in the code line below to debug your script.
+    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+
+
+# Press the green button in the gutter to run the script.
+if __name__ == '__main__':
+    print_hi('PyCharm')
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
```
Comment (tomaszsobczak92) [inline main.py]: I believe we can delete this file. This is just a pycharm boilerplate, probably not needed to run the tests

File: pages/login_page.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+from locators.consent_page_locators import ConsentPageLocators
+from locators.login_page_locators import LoginPageLocators
+
+test_url = "https://allegro.pl.allegrosandbox.pl/logowanie"
```
Comment (tomaszsobczak92) [inline pages/login_page.py]: Can we move variable into class definition? And there we will left two empty lines

**Review by tomaszsobczak92 — CHANGES_REQUESTED** [review]
Please, check my comments and amend them accordingly if you agree.

File: locators/home_page_locators.py (hunk 1 above)
Comment (paulinakujawa91) [reply locators/home_page_locators.py:2]: I changed this

File: main.py (hunk 2 above)
Comment (paulinakujawa91) [reply main.py]: I deleted it

File: pages/login_page.py (hunk 3 above)
Comment (paulinakujawa91) [reply pages/login_page.py]: I changed it

**Pull request merged**